content/artifacts/highlight-active-artifact-planets/plugin.js

```diff
@@ -0,0 +1,173 @@
+// Display the planet with active artifact, specially for cannon.
```

Comment:
from the code i can see that a special ring will be drawn if a photoid cannon is activated but not yet ready? is that true?  could you add a sentence here that explains that?

Reply:
Yes, it's true.
When cannon first activated, only a thin circle can be seen.
As time goes by, the thick arc will become bigger and bigger.
The thick arc become a circle when the cannon ready.